runtime/src/accounts_index.rs

```diff
@@ -212,9 +216,16 @@ impl AccountMapEntryMeta {
 }
 
 #[derive(Debug, Default)]
+/// one entry in the in-mem accounts index
+/// key is Pubkey
+/// this is the value
```

Comment:
nit: "Represents the value for an account key in the in-memory accounts index "

Reply:
updated